pkg/testgridanalysis/testgridconversion/ocp_synthentic_tests.go

```diff
@@ -36,10 +36,10 @@ func (openshiftSyntheticManager) CreateSyntheticTests(rawJobResults testgridanal
 			}
 
 			syntheticTests := map[string]*synthenticTestResult{
-				testgridanalysisapi.InstallTestName:             &synthenticTestResult{name: testgridanalysisapi.InstallTestName},
-				testgridanalysisapi.InstallTimeoutTestName:      &synthenticTestResult{name: testgridanalysisapi.InstallTestName},
-				testgridanalysisapi.InfrastructureTestName:      &synthenticTestResult{name: testgridanalysisapi.InfrastructureTestName},
-				testgridanalysisapi.FinalOperatorHealthTestName: &synthenticTestResult{name: testgridanalysisapi.FinalOperatorHealthTestName},
+				testgridanalysisapi.InstallTestName:             {name: testgridanalysisapi.InstallTestName},
+				testgridanalysisapi.InstallTimeoutTestName:      {name: testgridanalysisapi.InstallTestName},
+				testgridanalysisapi.InfrastructureTestName:      {name: testgridanalysisapi.InfrastructureTestName},
+				testgridanalysisapi.FinalOperatorHealthTestName: {name: testgridanalysisapi.FinalOperatorHealthTestName},
```

Comment:
this seems to be formatting my editor is doing....if it bugs you i'll remove it but i don't really feel like fighting with it.

Reply:
nm, i sorted it.